letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -15,10 +15,14 @@ set -e  # Work even if somebody does "sh thisscript.sh".
 
 # Note: you can set XDG_DATA_HOME or VENV_PATH before running this script,
 # if you want to change where the virtual environment will be installed
-XDG_DATA_HOME=${XDG_DATA_HOME:-~/.local/share}
+if [ -z "${XDG_DATA_HOME}" ]; then
+  XDG_DATA_HOME="~/.local/share"
+fi
```

Comment:
Use `"$VAR"` rather than `$VAR` for consistency with the rest of the script? (the funny `${VAR:-default}` syntax required the curly braces.

You could even go further and keep this on one line, though I don't really know if it's better:

`
[ -z "$XDG_DATA_HOME" ] && XDG_DATA_HOME=~/.local/share
`